docs/static/releasenotes.asciidoc

```diff
@@ -82,6 +83,26 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-17-5]]
+=== Logstash 8.17.5 Release Notes
+
+* The `SettingString` setting is now correctly non-nullable and `SettingNullableString` is nullable. https://github.com/elastic/logstash/pull/17522[#17522]
```

Comment:
Check out ["Release notes" section](https://github.com/elastic/logstash/pull/17522) in the PR. The developers put suggestions for user-focused release notes there, and they're usually more informative than the PR title.

Reply:
Thanks, i had seen that but it didnt quite make sense to me so I tried to make it a bit more explicit. Would you prefer that just gets copied over?

> Ensures that SettingString is correctly non-nullable and SettingNullableString is nullable. https://github.com/elastic/logstash/pull/17522[#17522]

Reply:
@donoghuc You make the call. I trust your judgement.  :-)